README.md

```diff
@@ -1093,12 +1096,16 @@ ANSWERS.addComponent('FilterOptions', {
       // The api field to filter on, configured on the Yext platform
       field: 'c_openNow',
       // The value for the above field to filter by
-      value: true
+      value: true,
+      // Whether this option will be selected on page load. Selected options stored in the url
+      // will take priority over appliedOnLoad. Defaults to false.
+      appliedOnLoad: false
```

Comment:
To confirm, there's no intelligent merging logic here, right? We don't try and reconcile the selected options in the url params with appliedOnLoad'd options, right? If there are options in the url params, that's all we show.

Reply:
yeah, was very careful about this.